tests/basictests/jupyterhub/jhtest.py

```diff
@@ -275,3 +320,4 @@ def check_exists_by_xpath(self, xpath):
     jhs = JHStress()
     jhs.run()
     jhs.quit()
+    
```

Comment:
Looks like a newline is needed here

Reply:
I see a newline there, maybe it took a bit for the diff to change

Reply:
There is probably a whitespace on that newline, so it is not really a "newline".

With the tests being so fragile, I'd say let's merge and fix this in next PR

Reply:
oh I see, okay got it.